lib/oci_utils/migrate/imgdevice.py

```diff
@@ -1381,19 +1249,13 @@ def get_os_release(self):
                 osdata = migrate_tools.exec_search('os-release', rootdir=mnt)
                 if osdata is not None:
                     with open(osdata, 'r') as f:
-                        osreleasedata = \
-                            [line.strip()
-                             for line in f.read().splitlines()
-                             if '=' in line]
-                    osdict = dict([re.sub(r'"', '', kv).split('=')
-                                   for kv in osreleasedata])
+                        osreleasedata = [line.strip() for line in f.read().splitlines() if '=' in line]
+                    osdict = dict([re.sub(r'"', '', kv).split('=') for kv in osreleasedata])
                     break
-                else:
-                    _logger.debug('os-release not found in %s' % mnt)
+                _logger.debug('os-release not found in %s', mnt)
         except Exception as e:
-            _logger.error('   Failed to collect os data: %s' % str(e),
-                          exc_info=True)
-        _logger.debug('os data: %s' % osdict)
+            _logger.error('   Failed to collect os data: %s', str(e), exc_info=True)
+        _logger.debug('os data: %s', osdict)
```

Comment:
Are you sure the stack trace is for the 'error' call , not the 'debug' one ?

Reply:
no, but we do not need the stack trace here at all, only the info it failed and why